requirements.txt

```diff
@@ -0,0 +1,15 @@
+Babel==2.5.1
```

Comment:
Why is Babel here? I'm confused why there are so many requirements in this file. This just seems unrelated.